Vagrantfile

```diff
@@ -10,7 +10,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   # please see the online documentation at vagrantup.com.
 
   # Every Vagrant virtual environment requires a box to build off of.
-  config.vm.box = "http://bit.ly/debian-wheezy-32-box"
+  config.vm.box = "https://github.com/kraksoft/vagrant-box-debian/releases/download/7.7.0/debian-7.7.0-amd64.box"
```

Comment:
Apparently this breaks the installation of Sphinx.

```
TASK [install_sphinx : Download Sphinx package] ********************************
fatal: [default]: FAILED! => {"changed": false, "dest": "/tmp/sphinx.deb", "failed": true, "msg": "Request failed", "response": "HTTP Error 404: Not Found", "state": "absent", "status_code": 404, "url": "http://sphinxsearch.com/files/sphinxsearch_2.2.9-release-1~wheezy_x86_64.deb"}
```

Correct URL would be: http://sphinxsearch.com/files/sphinxsearch_2.2.9-release-1~wheezy_amd64.deb.